packages/app/src/utils/createStoreHistory.ts

```diff
@@ -181,18 +184,30 @@ export function createStoreHistory<T extends object>([store, setStore]: [
     })
   }
 
+  function wrapIntoUndoing(fn: () => void) {
```

Comment:
Maybe we can have some short comment what is going on here :D

Reply:
Will do in a separate MR